packages/web-service/src/pages/habitat/work-start/habitat-work-start.js

```diff
@@ -1,6 +1,117 @@
+import Joi from 'joi'
 import pageRoute from '../../../routes/page-route.js'
 import { habitatURIs } from '../../../uris.js'
 
+const badgerLicenceSeasonOpen = `05-01-${new Date().getFullYear()}` // 1st May
```

Comment:
In the closed season , should this display the next season, so for instance Dec 2022 displays May 23 -> Nov 23 etc?